spark/src/main/java/org/apache/iceberg/spark/source/Writer.java

```diff
@@ -102,6 +102,10 @@
     this.replacePartitions = replacePartitions;
     this.applicationId = applicationId;
     this.wapId = wapId;
+
+    long tableTargetFileSize = Long.parseLong(table.properties().getOrDefault(
```

Comment:
What about using `PropertyUtil.propertyAsLong`?

Reply:
Ah, didn't know about that util class. Will fix.

Reply:
Fixed.